package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fpjs-incubator/botd-agent",
+  "name": "@fingerprintjs/botd",
   "version": "0.1.24",
```

Comment:
Don't forget to change the version here too. It should be `1.0.0-beta.0` I suppose.